source/fe/fe_raviart_thomas.cc

```diff
@@ -248,6 +248,173 @@ FE_RaviartThomas<dim>::initialize_support_points(const unsigned int deg)
 }
 
 
+template <int dim>
+void
+FE_RaviartThomas<dim>::initialize_quad_dof_index_permutation_and_sign_change()
+{
+  // For 1D do nothing.
+  //
+  // TODO: For 2D we simply keep the legacy behavior for now. This should be
+  // changed in the future and can be taken care of by similar means as the 3D
+  // case below. The legacy behavior can be found in fe_poly_tensor.cc in the
+  // function internal::FE_PolyTensor::get_dof_sign_change_h_div(...)
```

Comment:
Ah, I see. Yes, I agree that this should be done, but it can wait for a later patch.

Reply:
Should I open an issue to document the TODOs for later patches?

Reply:
As a general rule, small patches are easiest to deal with. You already see how long it took to get this one finished and reviewed, so yes: If you can break a patch into smaller, self-contained pieces, that is always preferrable.

Maybe as a remark on the side: This patch also contains a number of pieces where you rename variables, or just clean up little stuff. The way I operate is that I commit these smaller things separately into the branch I currently work on, and then create a number of branches based on `master` onto which I cherry-pick these clean-up patches and submit as separate pull requests. It is not difficult to get these sorts of things merged relatively quickly, and then the large patch you're working on becomes smaller and easier to get reviewed.